src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final int basisOne;
+    private final int basisTwo;
+    private final int width;
+
+    public IsoscelesTrapezoid(int basisOne, int basisTwo, int width, String color) {
+        this.basisOne = basisOne;
+        this.basisTwo = basisTwo;
+        this.width = width;
+        this.setColor(color);
+    }
+
+    @Override
+    public double getArea() {
+        return (double) ((basisOne + basisTwo) / 2) * width;
```

Comment:
All magic numbers in your code should be constants.
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names